data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Ванхорн", "Vanhorn" ],
+      [ "Ватсон", "Watson" ],
+      [ "Ватт", "Watt" ],
+      [ "Вейр", "Ware" ],
+      [ "Джолли", "Jolly" ],
+      [ "Хэм", "Ham" ],
+      [ "Виггинс", "Wiggins" ],
+      [ "Видаль", "Vidal" ],
+      [ "Виджид", "Vigil" ],
+      [ "Виейра", "Vieira" ],
+      [ "Вишневский", "Wisniewski" ],
+      [ "Дэнт", "Dent" ],
+      [ "Волк", "Wolf" ],
+      [ "Уилл", "Will" ],
+      [ "Вомак", "Womack" ],
+      [ "Гейтс", "Gates" ],
+      [ "Ист", "East" ],
+      [ "Хармс", "Harms" ],
+      [ "Вульф", "Wolfe" ],
+      [ "Вутен", "Wooten" ],
+      [ "Вьюер", "Weaver" ],
+      [ "Вэлш", "Welsh" ],
+      [ "Уайатт", "Wyatt" ],
+      [ "Габриэль", "Gabriel" ],
+      [ "Гавани", "Havens" ],
+      [ "Галант", "Gallant" ],
+      [ "Галлардо", "Gallardo" ],
+      [ "Галлоуэй", "Galloway" ],
+      [ "Гальвез", "Galvez" ],
+      [ "Ганье", "Gagne" ],
+      [ "Гарза", "Garza" ],
+      [ "Гарланд", "Garland" ],
+      [ "Гарретт", "Garrett" ],
+      [ "Гейгера", "Geiger" ],
+      [ "Гейл", "Gale" ],
+      [ "Герра", "Guerra" ],
+      [ "Гиллиленд", "Gilliland" ],
+      [ "Чейз", "Chase" ],
+      [ "Говард", "Howard" ],
+      [ "Гойнс", "Goins" ],
+      [ "Хэд", "Head" ],
+      [ "Гольдберг", "Goldberg" ],
+      [ "Гонсалвес", "Gonsalves" ],
+      [ "Кук", "Cook" ],
+      [ "Граббс", "Grubbs" ],
+      [ "Грейвс", "Graves" ],
+      [ "Грейнджер", "Granger" ],
+      [ "Гриер", "Grier" ],
+      [ "Грифон", "Griffin" ],
+      [ "Гроувер", "Grover" ],
+      [ "Грэм", "Graham" ],
+      [ "Гуахардо", "Guajardo" ],
+      [ "Гузман", "Guzman" ],
+      [ "Гундерсон", "Gunderson" ],
+      [ "Гэлвин", "Galvin" ],
+      [ "Давенпорт", "Davenport" ],
+      [ "Даглинг", "Darling" ],
+      [ "Дайки", "Dykes" ],
+      [ "Данжело", "Dangelo" ],
+      [ "Данэм", "Dunham" ],
+      [ "Дарби", "Darby" ],
+      [ "Дасилва", "Dasilva" ],
+      [ "Дауди", "Dowdy" ],
+      [ "Дворяне", "Nobles" ],
+      [ "Деврис", "Devries" ],
+      [ "Дежесус", "Dejesus" ],
+      [ "Делакрус", "Delacruz" ],
+      [ "Делароза", "Delarosa" ],
+      [ "Демерс", "Demers" ],
+      [ "День", "Day" ],
+      [ "дерево", "Wood" ],
+      [ "Джайлз", "Giles" ],
+      [ "Джейкобсон", "Jacobson" ],
+      [ "Джеймисон", "Jamison" ],
+      [ "Джентайл", "Gentile" ],
+      [ "Джентри", "Gentry" ],
+      [ "Джии", "Gee" ],
+      [ "Джилетт", "Gillette" ],
+      [ "Джилл", "Gill" ],
+      [ "Джуэлл", "Jewell" ],
+      [ "Джёрман", "German" ],
+      [ "Диван", "Couch" ],
+      [ "Диггс", "Diggs" ],
+      [ "Диль", "Diehl" ],
+      [ "Дион", "Dion" ],
+      [ "Длинный", "Long" ],
+      [ "Дов", "Dove" ],
+      [ "Дозье", "Dozier" ],
+      [ "Дол", "Dale" ],
+      [ "Дом", "House" ],
+      [ "Доти", "Doty" ],
+      [ "Дравер", "Driver" ],
+      [ "Дрейк", "Drake" ],
+      [ "Дрейпер", "Draper" ],
+      [ "Друг", "Friend" ],
+      [ "Дуайер", "Dwyer" ],
+      [ "Дубэ", "Dube" ],
```

Comment:
Given that the name is likely of French origin, it should probably be `Дюбе`, as in [Жан Дюбе](https://ru.wikipedia.org/wiki/%D0%94%D1%8E%D0%B1%D0%B5,_%D0%96%D0%B0%D0%BD), a French pianist.

However, it's worth noting that it could also be of South African or Indian origin. I was unable to find a reliable source for how to pronounce those.